README.md

```diff
@@ -94,6 +94,59 @@ docker run -d -p 4200:80 openmf/web-app:latest
 
 Access the webapp on http://localhost:4200 in your browser.
 
+### Docker compose
+It is possible to do a 'one-touch' installation of Mifos X Web App using containers (AKA "Docker").
+Fineract now packs the mifos community-app web UI in it's docker deploy.
+
+As Prerequisites, you must have `docker` and `docker-compose` installed on your machine; see
+[Docker Install](https://docs.docker.com/install/) and
+[Docker Compose Install](https://docs.docker.com/compose/install/).
+
+Now to run a new MifosX Web App instance you can simply:
+
+1. `git clone https://github.com/openMF/web-app.git ; cd web-app`
+1. for windows, use `git clone https://github.com/openMF/web-app.git --config core.autocrlf=input ; cd web-app`
+1. `docker-compose up -d`
+1. Access the webapp on http://localhost:4200 in your browser.
+
+You can also setup different confiurations for the MifosX Web App using environment variables:
+
+1. Use environment variables (best choice if you run with Docker Compose):
+
+Fineract backend settings
+```
+FINERACT_API_URLS
+```
+Value to set a Fineract server list (environments) to be used, Default value:
```

Comment:
I don't get the point of this list. I mean I know there's a dropdown on the login screen but why? Maybe for local testing it could be useful but for a regular deployment that should simply be hidden and not changeable to the user. I think let's just get rid of this configuration possibility or smthing.